backend/app/Http/Controllers/StackController.php

```diff
@@ -12,20 +12,31 @@ class StackController extends Controller
 {
     public function addStack(StackRequest $request): JsonResponse
     {
-        $data = $request->all();
+        $name = $request->name;
```

Comment:
why use form request here when you still declaring variable name here?

Reply:
you don't need to start writing $request->name again. just request->all() since the rules has been set in the form request.. or what do you feel?

Reply:
Will the $request->all() contain the user_id?

Reply:
yes